language-design.md

```diff
@@ -59,50 +72,45 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+It certainly highlights the process of identifying tradeoffs when attempting to optimize for multiple design goals.  This is noticeable in the tension between familiarity/precision (using `RGB(x,x,x)` for a particular level of grey) and more specific but less precise name (`gray(x)` (does a higher value mean lighter or darker?)). 
 
+The comments are also quite interesting, as we see that one individual suggests using the common `greyscale()` term to name the function. Although it is not inherently any clearer than `gray()`, people are used to seeing and using it. However, as we can see from a later commenter,the function `greyscale()` is already defined by [Filter Effects](https://drafts.fxtf.org/filters/#grayscaleEquivalent). This namespace conflict might suggest suboptimal design as well.
```

Comment:
I would argue that we don't want both `greyscale` and `grayscale`, for much the same reason that an entirely NL-based language is going to be hard to use, at best.